log/log.go

```diff
@@ -1,72 +1,108 @@
 package log
 
 import (
-	"fmt"
-
-	"github.com/golang/glog"
+	"github.com/rs/zerolog"
+	"github.com/rs/zerolog/log"
+	"os"
```

Comment:
trivial - the imports in the kiali src code base are formatted specifically - base libs first, blank line, third party libs, blank line, kiali libs last. So this should be:

```suggestion
	"os"

	"github.com/rs/zerolog"
	"github.com/rs/zerolog/log"
```